src/main/java/games/virus/VirusGameState.java

```diff
@@ -4,6 +4,7 @@
 import core.AbstractGameState;
 import core.components.Component;
 import core.components.Deck;
+import core.components.PartialObservableDeck;
```

Comment:
Strange to add an import here with no other code, was it missing?

Reply:
Odd. Removed.